charters/UploadService/charter.md

```diff
@@ -0,0 +1,78 @@
+# Upload Service
+
+## Description
+The Upload Service enables fast, cost-effective upload, staging, and validation of data submitted to the Human Cell Atlas (HCA) Data Coordination Platform (DCP) from across the world.
+
+## Objectives
+1. Enable submitters to upload files quickly and cost-effectively to temporary *staging areas* for subsequent submission to the Data Storage Service (DSS)
+1. Provide tools a command line interface (CLI) and APIs to upload files to staging areas
+1. Provide DCP developers the ability to validate files loaded into staging areas with user-defined validation jobs
+
+## In-scope
+* Provide staging areas that enable fast and cost-effective data transfer between:
+    * The DCP Ingestion UI workflow and the DSS
+    * The DCP Ingestion *data wrangler* workflow and the DSS
```

Comment:
UI workflow and data wrangler workflows are essentially the same thing. You can collapse them into a single DCP ingestion workflow.

Transfer between a workflow and the DSS doesn't seem to make sense. How about "Enables data file upload and release to the DSS upon completion of the ingestion workflow"?